### PR TITLE
minor bug fix

### PR DESCRIPTION
**Pull request by dkasb**

### CODE REVIEW

File: includes/UserAdd.php [hunk 1]
```diff
@@ -102,7 +102,8 @@
 	}
 
 	$hash = password_hash( $_POST['passwd'] , PASSWORD_BCRYPT );
-
+
```
Comment (pkgodara) [inline includes/UserAdd.php:105]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

File: includes/UserAdd.php [hunk 2]
```diff
@@ -102,7 +102,8 @@
 	}
 
 	$hash = password_hash( $_POST['passwd'] , PASSWORD_BCRYPT );
-
+	
+	$flag = 0;
```
Comment (pkgodara) [inline includes/UserAdd.php:106]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

File: includes/UserAdd.php [hunk 3]
```diff
@@ -115,6 +116,12 @@
 		echo "<h1 style ='color:red'>Error updating database information, Try another username because same user name already exists</h1>";
 		die();
 	}
+
```
Comment (pkgodara) [inline includes/UserAdd.php:119]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

File: includes/UserAdd.php [hunk 4]
```diff
@@ -115,6 +116,12 @@
 		echo "<h1 style ='color:red'>Error updating database information, Try another username because same user name already exists</h1>";
 		die();
 	}
+	
+	else
```
Comment (pkgodara) [inline includes/UserAdd.php:120]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Expected 1 space after ELSE keyword; newline found

File: includes/UserAdd.php [hunk 5]
```diff
@@ -115,6 +116,12 @@
...
+	{
```
Comment (pkgodara) [inline includes/UserAdd.php:121]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

File: includes/UserAdd.php [hunk 6]
```diff
@@ -115,6 +116,12 @@
...
+		$flag = 1 ;
```
Comment (pkgodara) [inline includes/UserAdd.php:122]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

File: includes/UserAdd.php [hunk 7]
```diff
@@ -115,6 +116,12 @@
...
+	}
```
Comment (pkgodara) [inline includes/UserAdd.php:123]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

File: includes/UserAdd.php [hunk 8]
```diff
@@ -115,6 +116,12 @@
...
+
```
Comment (pkgodara) [inline includes/UserAdd.php:124]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

File: includes/UserAdd.php [hunk 9]
```diff
@@ -147,11 +154,18 @@
 
 	if( ! $stmt->execute() )
 	{
-		//echo "Error : $sqlConn->errno : $sqlConn->error <br>";
-		echo "Error creating database for user, contact Admin";
+
```
Comment (pkgodara) [inline includes/UserAdd.php:157]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

File: includes/UserAdd.php [hunk 10]
```diff
@@ -147,11 +154,18 @@
 
 	if( ! $stmt->execute() )
 	{
-		//echo "Error : $sqlConn->errno : $sqlConn->error <br>";
-		echo "Error creating database for user, contact Admin";
+		
+		if($flag == 1)
```
Comment (pkgodara) [inline includes/UserAdd.php:158]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Expected 1 space after IF keyword; 0 found
* Expected 1 space after closing parenthesis; found 5

File: includes/UserAdd.php [hunk 11]
```diff
@@ -147,11 +154,18 @@
...
+	{
```
Comment (pkgodara) [inline includes/UserAdd.php:159]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Line indented incorrectly; expected at least 8 spaces, found 4

File: includes/UserAdd.php [hunk 12]
```diff
@@ -147,11 +154,18 @@
...
+		echo "User Successfully created. ";
```
Comment (pkgodara) [inline includes/UserAdd.php:160]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Line indented incorrectly; expected at least 12 spaces, found 8

File: includes/UserAdd.php [hunk 13]
```diff
@@ -147,11 +154,18 @@
...
+		die();
```
Comment (pkgodara) [inline includes/UserAdd.php:161]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Line indented incorrectly; expected at least 12 spaces, found 8

File: includes/UserAdd.php [hunk 14]
```diff
@@ -147,11 +154,18 @@
...
+	}
```
Comment (pkgodara) [inline includes/UserAdd.php:162]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Line indented incorrectly; expected 8 spaces, found 4
* Closing brace indented incorrectly; expected 8 spaces, found 4
* Expected 1 space after closing brace; newline found
* Whitespace found at end of line

File: includes/UserAdd.php [hunk 15]
```diff
@@ -147,11 +154,18 @@
...
 	{
-		//echo "Error : $sqlConn->errno : $sqlConn->error <br>";
-		echo "Error creating database for user, contact Admin";
+		
+		if($flag == 1)
+	{
+		echo "User Successfully created. ";
+		die();
+	}	
+	else
```
Comment (pkgodara) [inline includes/UserAdd.php:163]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Line indented incorrectly; expected 8 spaces, found 4
* Expected 1 space after ELSE keyword; newline found

File: includes/UserAdd.php [hunk 16]
```diff
@@ -147,11 +154,18 @@
...
+	{
```
Comment (pkgodara) [inline includes/UserAdd.php:164]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Line indented incorrectly; expected at least 8 spaces, found 4

File: includes/UserAdd.php [hunk 17]
```diff
@@ -147,11 +154,18 @@
...
+		echo "Error creating database for user, contact Admin ";
```
Comment (pkgodara) [inline includes/UserAdd.php:165]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Line indented incorrectly; expected at least 12 spaces, found 8

File: includes/UserAdd.php [hunk 18]
```diff
@@ -147,11 +154,18 @@
...
 		die();
 	}
-	
+	}
```
Comment (pkgodara) [inline includes/UserAdd.php:168]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

File: includes/UserAdd.php [hunk 19]
```diff
@@ -183,11 +197,10 @@
 
 	if( ! $stmt->execute() )
 	{
-		//echo "Error : $sqlConn->errno : $sqlConn->error <br>";
 		echo "Error in creating count database for user, contact Admin.";
 		die();
 	}
-
+
```
Comment (pkgodara) [inline includes/UserAdd.php:203]: __[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

**Comment by pkgodara** [comment]
Click here if you want to create code_sniffer's auto-correct Pull Request
[Auto Correct by SideCI](https://sideci.com/gh/80422762/issues/pull_request/67/diff/304006d9f071f3ff421460b13408b395a17ba758?open_modal=1#code_sniffer)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-code_sniffer)

**Review by NabeelQaiser — APPROVED** [review]

**Pull request merged**